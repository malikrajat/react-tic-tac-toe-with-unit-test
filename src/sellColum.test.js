import React from 'react';
import { shallow, configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import App from './sellColum';

configure({ adapter: new Adapter() });

describe("App component test ", () => {

  it("Find class name by shallow method", () => {
    const wrapper = shallow(<SellColum row={0} col={0} turn="X" setTurn={changeTurn}/>);
    expect(wrapper.hasClass('sell')).toEqual(true);
  });

  it('sellColum snapshot renders', () => {
    const component = renderer.create(<sellColum/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});