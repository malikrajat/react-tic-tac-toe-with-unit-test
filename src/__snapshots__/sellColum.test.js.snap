// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component test  sellColum snapshot renders 1`] = `<sellColum />`;
